tsl/src/continuous_aggs/create.c

```diff
@@ -1460,7 +1473,7 @@ finalizequery_get_select_query(FinalizeQueryInfo *inp, List *matcollist,
 	 * modify this to reflect the materialization table we just
 	 * created.
```

Comment:
could you please add a comment here on when there are more than 1 entries here. The comment on line 1472 is no longer accurate and needs to be updated.